FSharp.AutoComplete/test/integration/TestHelpers.fsx

```diff
@@ -81,9 +84,10 @@ let writeNormalizedOutput (fn: string) (s: string) =
         lines.[i] <- Regex.Replace(lines.[i].Replace('\\','/'),
                                    "[a-zA-Z]:/.*?FSharp.AutoComplete/test/(.*?(\"|$))",
                                    "<absolute path removed>/test/$1")
-
-    if Path.GetExtension fn = ".json" then
-      lines.[i] <- formatJson lines.[i]
+    
+    // This throws because each line is not valid json by itself
+    //if Path.GetExtension fn = ".json" then
+    //  lines.[i] <- formatJson lines.[i]
```

Comment:
Each line *should* be valid json. Did this happen before you changed `Formatting.Indented` above?